mathesar/tests/api/test_column_api_display_options.py

```diff
@@ -307,6 +312,8 @@ def test_column_alter_same_type_display_options(
     column_index = 2
     column = columns[column_index]
     pre_alter_display_options = column.display_options
+    print("----------------------------------------------------")
```

Comment:
Please remove `print` statements, We use `print` statements only to output messages to the user.

Reply:
Oh gosh, that's embarrassing 🙄. Thanks for catching! I've fixed it in f98ad2eabc9da427bf7fc9e02764860d9a7cb740